### PR TITLE
Update logging formatting

### PR DESCRIPTION
**Pull request by joshua-plautz**
* Add a prefix to all log messages based on what step is being executed
* Modify threading to bring log watching back into the rest of the process
* standardize all logging messages for width and format
* Shorten info/error/etc prefixes to 4 characters to keep columns intact

### CODE REVIEW

File: iuf [hunk 1]
```diff
@@ -259,11 +259,15 @@ def process_activity(config):
...
+    install_logger.info(f"[ACTIVITY: {config.activity.name:47}] BEG Install started at {config.stages.installer_start}")
     config.activity.create_activity()
     config.activity.run_stages(resume=False)
 
     config.activity.state({"state": "waiting_admin"})
-    install_logger.info("Install completed in {}".format(elapsed_time(config.stages.installer_start)))
+    duration = elapsed_time(config.stages.installer_start)
+    install_logger.info(f"[ACTIVITY: {config.activity.name:47}] END Completed in {duration}")
+
+    #install_logger.info("Install completed in {}".format(elapsed_time(config.stages.installer_start)))
```
Comment (larry-tordsen) [inline iuf]: Minor: I'd delete the commented out `install_logger.info` problem.

File: iuf [hunk 2]
```diff
@@ -771,8 +778,11 @@ def main():
 
     try:
         args.func(config)
-    except AttributeError:
-        install_logger.error("A subcommand was not specified.")
+    except AttributeError as e:
```
Comment (larry-tordsen) [inline iuf:784]: Nice change.  These lines were annoying because they masked other `AttributeError`'s.

**Review by larry-tordsen — APPROVED** [review]

**Comment by deantroe** [comment]
If it hasn't been done already, please create a separate Jira for us to update the CSM docs with this new ouput format for any examples in there.

File: lib/SiteConfig.py [hunk 3]
```diff
@@ -284,11 +283,7 @@ def manage_session_vars(self, session_vars, write=False):
                 yaml.dump(self.session_vars, fhandle)
 
         with open(self.sv_path, "w") as fhandle:
-            if not self.printed_sv_msg:
-                install_logger.info("Dumping rendered site variables to {}".format(self.sv_path))
-                self.printed_sv_msg = True
-            else:
-                install_logger.debug("Dumping rendered site variables to {}".format(self.sv_path))
+            install_logger.debug("Dumping rendered site variables to {}".format(self.sv_path))
```
Comment (JohnNavitskyHPE) [inline lib/SiteConfig.py:286]: How about we try to add some context to this message, like "Saving rendered site variables to be used with sat bootprep run --vars-file, to {}"

**Review by JohnNavitskyHPE — APPROVED** [review]

**Pull request merged**